desktop/src/main/java/bisq/desktop/bisq.css

```diff
@@ -31,6 +31,10 @@
     -fx-text-fill: -bs-rd-error-red;
 }
 
+.text-gray-ddd {
```

Comment:
I think it is better not to include specific color codes in the css class name as it might be different colors in different themes.